client/static/post.js

```diff
@@ -0,0 +1,88 @@
+const url = window.location.href;
+
+const postTitle = document.getElementById('postTitle');
+const postName = document.getElementById('postName');
+const postStory = document.getElementById('postStory');
+
+if(url.includes('#')) {
+    const id = url.split('#').pop();
+    renderPost(id);
+} else {
+    let publishForm = document.getElementById('publishForm');
+    publishForm.addEventListener('submit', postContent)
+}
+
+async function postContent(e) {
+    e.preventDefault();
+    const postContent = {
+        title: e.target.postTitle.value,
+        name: e.target.postName.value,
+        story: e.target.postStory.value
+    };
+
+    const options = {
+        method: 'POST',
+        body: JSON.stringify(postContent),
+        headers: { "Content-Type": "application/json" }
+    };
+
+    const response = await fetch('http://localhost:3000/', options)
+    const json = await response.json()
+    console.log(json);
+    console.log(postContent);
```

Comment:
A small point but get rid of any unnecessary code like any console.log. Helps your website's performance as well 👍